swagger/src/main/scala/org/http4s/rho/swagger/package.scala

```diff
@@ -95,6 +95,8 @@ package object swagger {
 
     def isTask: Boolean =
       t <:< typeOf[Task[_]]
-  }
 
+    def isSwaggerFile: Boolean =
+      t <:< typeOf[SwaggerFileResponse[_]]
```

Comment:
Interesting! I wonder why that could be. I don't think it matters for the purposes of this PR.